File: devices-api-java-mvc/src/main/java/com/lucas/devicesapijavamvc/api/dto/DeviceResponse.java
package com.lucas.devicesapijavamvc.api.dto;

import java.time.OffsetDateTime;
import java.util.UUID;

public record DeviceResponse(
        Long id,
        UUID externalId,
        String name,
        String brand,
        String state,
        OffsetDateTime creationTime
) {}

File: devices-api-kotlin-reactive/src/main/kotlin/com/lucas/devicesapikotlinreactive/infrastructure/api/dto/DeviceResponse.kt
package com.lucas.devicesapikotlinreactive.infrastructure.api.dto

import java.time.OffsetDateTime
import java.util.UUID

data class DeviceResponse(
    val id: Long,
    val externalId: UUID,
    val name: String,
    val brand: String,
    val state: String,
    val creationTime: OffsetDateTime
)
